mm2src/coins_activation/src/platform_coin_with_tokens.rs

```diff
@@ -68,7 +69,10 @@ pub trait TokenInitializer {
 
     async fn enable_tokens(
         &self,
-        params: Vec<TokenActivationParams<Self::TokenActivationRequest, Self::TokenProtocol>>,
+        params: Vec<(
+            Json,
+            TokenActivationParams<Self::TokenActivationRequest, Self::TokenProtocol>,
+        )>,
```

Comment:
Makes sense. I didn't do that because the configuration JSON comes from the coins file, while the others come from requests. I also got confused due to the naming, thinking that it was related to request activation parameters. However, `TokenActivationParams` is an internal type meaning all parameters needed for coin activation and not related to requests. Fixed.